### ISSUE TITLE
Question about LoRA settings

### ISSUE THREAD

**Issue by ZQL810** [zjunlp/unlearn#10]
When reproducing GA_GDR_SURE (Table 1), I accidentally used r=32, alpha=32, dropout=0.05 (from code), and I could reproduce Table 1 correctly.But when I used the paper’s stated unified LoRA: r=8, alpha=16, dropout=0.1, it only achieved KFR ≈ 0.58.Due to limited GPU memory, I set max_length to 128.I am not sure if I have mistakenly modified the code somewhere.
Could you clarify: Which LoRA setting is official for baseline models and relearn? Is the r=8, alpha=16, dropout=0.1 setup only for ReLearn rather than baselines?

Thank you!

<img width="1432" height="1032" alt="Image" src="https://github.com/user-attachments/assets/6f5a00b5-ddb6-4bf7-a7cc-00b7c7a91b55" />

<img width="1235" height="1018" alt="Image" src="https://github.com/user-attachments/assets/e6d46875-22c9-443d-a6cd-36101a03241e" />

<img width="1193" height="977" alt="Image" src="https://github.com/user-attachments/assets/feaee578-c5f6-42c5-96b0-af920637256f" />

<img width="1368" height="1028" alt="Image" src="https://github.com/user-attachments/assets/702b9ede-3fa4-4fc4-bbf0-bea770fef2de" />

**Comment by HaomingX** [comment]
Hi, I think this behavior is likely influenced by multiple factors, including hyperparameters and even the training hardware. In general, training an unlearning baseline involves an inherent trade-off. Extending the training duration or increasing the learning rate can improve KFR, but this often comes at the cost of a significant drop in KRR.

**Comment by ZQL810** [comment]
> Hi, I think this behavior is likely influenced by multiple factors, including hyperparameters and even the training hardware. In general, training an unlearning baseline involves an inherent trade-off. Extending the training duration or increasing the learning rate can improve KFR, but this often comes at the cost of a significant drop in KRR.

Hi! Thank you for your response.
I also previously attempted to reproduce NPO_GDR_SURE. Under these two sets of configurations, the performance of KFR did not reach the results reported in the paper, and there is a significant discrepancy in the ROUGE_RECALL_F scores(0.04 in the paper).(The first picture is the results under lora(r=32 alpha=32 dropout=0.05)
so,if I were to use this same set of configurations(r=32 alpha=32 dropout=0.05) for all my experiments, should I increase the learning rate to achieve the results of the comparative experiment shown in Table 1 of the paper?
I would greatly appreciate it if you could answer my questions. 



<img width="1241" height="1021" alt="Image" src="https://github.com/user-attachments/assets/20bea353-41a0-4c31-82be-b03547a9eb55" />

<img width="1223" height="971" alt="Image" src="https://github.com/user-attachments/assets/b9d7d24c-c22d-4f89-8e8b-48bf4fc10b66" />

**Comment by HaomingX** [comment]
Yes, I think you can try to increase the `epoch` or the `learning rate`.

**Comment by ZQL810** [comment]
> Yes, I think you can try to increase the `epoch` or the `learning rate`.

Okay, thank you very much for your reply.
